.github/workflows/pr-review-advisor-repair-reconcile.yaml

```
@@ -0,0 +1,221 @@
+            /tools/pr-review-advisor-repair/
+          sparse-checkout-cone-mode: false
+
+      - name: Setup Node
+        uses: actions/setup-node@820762786026740c76f36085b0efc47a31fe5020 # v7.0.0
+        with:
+          node-version: "22"
+
+      - name: Wait for the exact required checks, CodeQL, and Advisor run
+        env:
+          ATTEMPT_KEY: ${{ needs.collect.outputs.attempt_key }}
+          BASE_SHA: ${{ needs.collect.outputs.base_sha }}
```

actionlint (1.7.12)

[error] property "base_sha" is not defined in object type {attempt_key: string; head_ref: string; validation_artifact_id: string}

(expression)